main/acle.md

```diff
@@ -4855,7 +4885,17 @@ one of the following is true:
 
 *   F shares ZT0 with its caller
 
-All other functions have a private-ZA interface.
+All other functions have either a private-ZA or an agnostic-ZA interface.
```

Comment:
I think we should leave this sentence as-is and describe the agnostic-ZA case before it.  I'd recomment restructuring the text as follows:

```
If a C or C++ function F forms part of the object code's ABI:

*   the object code function has a shared-ZA interface if and only if at least
    one of the following is true:

    *   F shares ZA with its caller

    *   F shares ZT0 with its caller

*   the object code function has an agnostic-ZA interface if and only if
    F's type has an `__arm_agnostic("sme_za_state")` attribute.

All other functions have either a private-ZA interface.
```